python/sdk/forevervm_sdk/repl.py

```diff
@@ -18,6 +18,7 @@ class ReplException(Exception):
 class ReplExecResult:
     _request_id = -1
     _instruction_id = -1
+    _output = deque[StandardOutput]()
```

Comment:
the changes here are related but came up during linting and minimal enough that I threw them into this PR